basics/editor-setup.md

```diff
@@ -23,37 +23,47 @@ The choice of an editor is a very personal thing. There are many
 editors, and most programmers have a favorite editor that they use for
```

Comment:
A few lines above it says that IDLE won't be used in this tutorial. Please edit it to say e.g. that you can use IDLE but you should probably explore other options as well.